Terasology/LightAndShadow
# feat: custom loading screen

### Purpose

Set custom loading screen for Light And Shadow.

### How to test

- Start a LAS game
- Notice the loading screen (sometimes it takes a few seconds to appear?)

#### Loading Screen

<img width="860" alt="Screenshot_13" src="https://user-images.githubusercontent.com/48293545/88971986-888fd180-d2bd-11ea-864f-86575fcd04d6.png">
